### PR TITLE
kdump: fix guest can not do kdump over ssh

### PR DESCRIPTION
**Pull request by liaolucy**
Change location of SSH connection setup;
Add check core dump for each panic;

bug id: 1543770
Signed-off-by: Chunyu Liao cliao@redhat.com

### CODE REVIEW

**Comment by liaolucy** [comment]
'1-Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.kdump.one_vm.with_ssh' Finshed 2018-08-31 02:20:53 2018-08-31 02:34:43 PASS 830.076786
'2-Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.kdump.one_vm.with_ssh.nmi' Finshed 2018-08-31 02:34:44 2018-08-31 02:38:40 PASS 236.768012

**Comment by liaolucy** [comment]
@xiangchunfu @ngu-niny please help review this patch, thanks.

**Comment by xiywang** [comment]
Seems the branch you are using is not clear which leads to commit log redundant

File: generic/tests/kdump.py [hunk 1]
```diff
@@ -148,13 +129,18 @@ def crash_test(test, vm, vcpu, crash_cmd, timeout):
     vmcore_rm_cmd = vmcore_rm_cmd % vmcore_path
     kdump_restart_cmd = vm.params.get("kdump_restart_cmd",
                                       "service kdump restart")
+    kdump_propagate_cmd = vm.params.get("kdump_propagate_cmd")
```
[the remark this answers is not in the document]
Comment (liaolucy) [reply generic/tests/kdump.py]: from kdump.cfg: 
rhel6: kdump_propagate_cmd = "service kdump propagate"
rhel7: kdump_propagate_cmd = "kdumpctl propagate"

[the remark this answers is not in the document]
Comment (xiywang) [reply generic/tests/kdump.py]: I mean, if you want to set default value, then you can use params.get("xx", "default value"). Otherwise, use params["xx"].

**Pull request closed without merging**